python/src/SobolIndicesAlgorithmImplementation_doc.i.in

```diff
@@ -138,6 +138,26 @@ the `getFirstOrderIndicesInterval` and `getTotalOrderIndicesInterval` methods. D
 and confidence level values are parametrized thanks to the resource map keys `SobolIndicesAlgorithm-BootstrapSize`
 and `SobolIndicesAlgorithm-BootstrapConfidenceLevel`. Appropriate setters
 might be called to change their values.
+
+
+References
+----------
+ - A. Saltelli, 2002, Making best use of model evaluations to compute
+   sensitivity indices, Computer Physics Communication, 145, 580–297
+ - I. M. Sobol, 1993, Sensitivity analysis for non-linear mathematical model,
+   Math. Modelling Comput. Exp., 1, 407–414
+ - I.M. Sobol, S. Tarantola, D. Gatelli, S.S. Kucherenko and W. Mauntz, 2007,
+   Estimating the approximation errors when fixing unessential factors in
+   global sensitivity analysis, Reliability Engineering and System Safety, 92,
+   957–960
+ - M.J.W. Jansen, 1999, Analysis of variance designs for model output,
+   Computer Physics Communication, 117, 35–43
+ - J-M. Martinez, 2011, Analyse de sensibilite globale par decomposition de la
+   variance, Presentation in the meeting of GdR
+   Ondes and GdR MASCOT-NUM, January, 13th, 2011, Institut Henri Poincare,
+   Paris, France
+ - Fabrice Gamboa, Alexandre Janon, Thierry Klein, Agnès Lagnoux. Sensitivity
+   analysis for multidimensional and functional outputs. 2013
```

Comment:
These should be in bibliography.rst, and each relevant algorithm implementation should referer to it.
Maybe we could also link to the pdf.